lighthouse-core/gather/gatherers/seo/tap-targets.js

```diff
@@ -249,6 +180,48 @@ function truncate(str, maxLength) {
   return str.slice(0, maxLength - 1) + '…';
 }
 
+/**
+ * @param {Element} el
+ * @param {{x: number, y: number}} elCenterPoint
+ */
+/* istanbul ignore next */
+function elementCenterIsAtZAxisTop(el, elCenterPoint) {
+  const viewportHeight = window.innerHeight;
+  const targetScrollY = Math.floor(elCenterPoint.y / viewportHeight) * viewportHeight;
+  if (window.scrollY !== targetScrollY) {
+    window.scrollTo(0, targetScrollY);
+  }
+
+  const topEl = document.elementFromPoint(
+    elCenterPoint.x,
+    elCenterPoint.y - window.scrollY
+  );
+
+  return topEl === el || el.contains(topEl);
+}
+
+/* istanbul ignore next */
+function disableFixedAndStickyElementPointerEvents() {
+  const className = 'lighthouse-disable-pointer-events';
+  const styleTag = document.createElement('style');
+  styleTag.innerHTML = `.${className} { pointer-events: none !important }`;
+  document.body.appendChild(styleTag);
+
+  Array.from(document.querySelectorAll('*')).forEach(el => {
```

Comment:
can just do
```suggestion
  document.querySelectorAll('*').forEach(el => {
```
since this code only ever runs in recent Chrome

Reply:
Oh, TIL. 👍